pegasus/sites.v3/advocacy.code.org/public/css/advocacy-home.css

```diff
@@ -15,7 +15,10 @@
   color: #666;
 }
 
-.info a:link, .info a:visited, .states-link a:link, .states-link a:visited {
```

Comment:
It looks like we've got a good bunch of linter updates yeah?

Reply:
I think there's just this one, and line 120(old)/173(new).